### PR TITLE
init documentation for `ServiceWorkerGlobalScope.cookiechange_event`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

init documentation for `ServiceWorkerGlobalScope.cookiechange_event`

### Additional details

working on https://developer.mozilla.org/en-US/docs/Web/API/Cookie_Store_API
reference to https://wicg.github.io/cookie-store/

### CODE REVIEW

File: files/en-us/web/api/serviceworkerglobalscope/cookiechange_event/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+slug: Web/API/ServiceWorkerGlobalScope/cookiechange_event
+page-type: web-api-event
+status:
+  - experimental
+browser-compat: api.ServiceWorkerGlobalScope.cookiechange_event
+---
+
+{{APIRef("Cookie Store API")}}{{SeeCompatTable}}
+
+The **`cookiechange`** event of the {{domxref("ServiceWorkerGlobalScope")}} interface is fired when any cookie changes occurs which match the service worker's cookie change subscription list added via {{domxref("CookieStoreManager.subscribe()")}}.
```
Comment (skyclouds2001) [inline files/en-us/web/api/serviceworkerglobalscope/cookiechange_event/index.md:13]:
```suggestion
The **`cookiechange`** event of the {{domxref("ServiceWorkerGlobalScope")}} interface is fired when any cookie changes occurs which match the service worker's cookie change subscription list.
```

File: files/en-us/web/api/serviceworkerglobalscope/cookiechange_event/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+slug: Web/API/ServiceWorkerGlobalScope/cookiechange_event
+page-type: web-api-event
+status:
+  - experimental
+browser-compat: api.ServiceWorkerGlobalScope.cookiechange_event
+---
+
+{{APIRef("Cookie Store API")}}{{SeeCompatTable}}
+
+The **`cookiechange`** event of the {{domxref("ServiceWorkerGlobalScope")}} interface is fired when any cookie changes occurs which match the service worker's cookie change subscription list.
```
Comment (skyclouds2001) [inline files/en-us/web/api/serviceworkerglobalscope/cookiechange_event/index.md:13]:
```suggestion
The **`cookiechange`** event of the {{domxref("ServiceWorkerGlobalScope")}} interface is fired when any cookie changes occur which match the service worker's cookie change subscription list.
```

File: files/en-us/web/api/serviceworkerglobalscope/cookiechange_event/index.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+status:
+  - experimental
+browser-compat: api.ServiceWorkerGlobalScope.cookiechange_event
+---
+
+{{APIRef("Cookie Store API")}}{{SeeCompatTable}}
+
+The **`cookiechange`** event of the {{domxref("ServiceWorkerGlobalScope")}} interface is fired when any cookie changes occur which match the service worker's cookie change subscription list.
+
+This event is not cancelable and does not bubble.
```
Comment (teoli2003) [inline files/en-us/web/api/serviceworkerglobalscope/cookiechange_event/index.md:15]:
```suggestion
The **`cookiechange`** event of the {{domxref("ServiceWorkerGlobalScope")}} interface is fired when a cookie change occurs that matches the service worker's cookie change subscription list.

This event is not cancelable and does not bubble.
```

File: files/en-us/web/api/serviceworkerglobalscope/index.md [hunk 4]
```diff
@@ -53,6 +53,8 @@ Listen to this event using {{domxref("EventTarget/addEventListener()", "addEvent
   - : Fired on a payment app's service worker to check whether it is ready to handle a payment. Specifically, it is fired when the merchant website calls {{domxref("PaymentRequest.PaymentRequest", "new PaymentRequest()")}}.
 - {{domxref("ServiceWorkerGlobalScope/contentdelete_event", "contentdelete")}} {{Experimental_Inline}}
   - : Occurs when an item is removed from the {{domxref("ContentIndex", "Content Index")}}.
+- {{domxref("ServiceWorkerGlobalScope/cookiechange_event", "cookiechange")}} {{Experimental_Inline}}
+  - : Fired when any cookie changes have occurred which match the Service Worker's cookie change subscription list.
```
Comment (teoli2003) [inline files/en-us/web/api/serviceworkerglobalscope/index.md:57]:
```suggestion
  - : Fired when a cookie change has occurred that matches the service worker's cookie change subscription list.
```

**Review by teoli2003 — APPROVED** [review]
A few nits that I fix and we can merge this. Thank you.

**Pull request merged**